core/templates/components/state-directives/rule-editor/rule-type-selector.directive.ts

```diff
@@ -46,25 +46,18 @@ angular.module('oppia').directive('ruleTypeSelector', [function() {
 
           var ruleTypesToDescriptions = INTERACTION_SPECS[
             StateInteractionIdService.savedMemento].rule_descriptions;
+          // TODO(bhenning): The order of choices should be meaningful. E.g.,
```

Comment:
Please could you file an issue for this (if there isn't already one), and have this TODO refer to the issue instead? It may be worth prioritizing it a bit for the LaCE team stuff -- I keep running into this issue.

EDIT: do you still need this if you're fixing it based on the comment thread [here](https://github.com/oppia/oppia/pull/10662/files#r487450156)? I think if we go with the suggestion in that comment, we don't need this any more -- having Equals as the default option should be sufficient.

Reply:
Done, fixed it here itself by checking for 'Equal' or 'Exactly' rules and always having them be first.